lib/gui/app.js

```diff
@@ -123,10 +125,10 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
 
     AnalyticsService.logEvent('Notifying update');
 
-    return UpdateNotifierService.notify();
-
+    return s3.getLatestVersion(s3.BUCKET_URL_PRODUCTION, {
```

Comment:
....what??

So we first call `isLatestVersion(CURRENT_VERSION)`, and if that returns false we then call `getLatestVersion()`? Which of course results in two separate S3 calls, and seems a bit inefficient to me.

Why not just have a single call  to `getLatestVersion()` at the top here, get rid of `isLatestVersion`, and compare the version here directly??

(I hope I'm not missing anything obvious for it being done in this 'inefficient' way!?!)